src/main/java/org/folio/list/services/CustomTenantService.java

```diff
@@ -37,7 +42,17 @@ protected void afterTenantUpdate(TenantAttributes tenantAttributes) {
     log.info("Initializing system user");
     prepareSystemUserService.setupSystemUser();
 
-    log.info("Verifying lists are up to date");
-    migrationService.verifyListsAreUpToDate();
+    // In Eureka, the system user often takes a short bit of time for its permissions to be assigned, so retry in the
+    // case of failures related to missing permissions
+    RetryTemplate.builder()
```

Comment:
Is there any way we can add a log message if it fails/times out to say it was not possible?

Reply:
Good call! That should be doable